presto-docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,92 @@
+====================
+Hudi connector
+====================
+
+.. contents::
```

Comment:
TOC is not included in connectors' doc source file. Let's remove it. The doc generator will automatically add one when rendering the page.

Reply:
done